demos/src/formatOptions.js

```diff
@@ -0,0 +1,39 @@
+const dates = document.querySelectorAll('[data-o-component="o-date"]');
+const now = new Date();
+const recently = new Date();
```

Comment:
We can change the setting of the dates onto the construction, it may aid in readability --
```js
const recently = new Date().setMinutes(now.getMinutes() - 23);
const today = new Date().setHours(now.getHours() - 6);
const yesterday = new Date().setHours(now.getHours() - 24);
const daysAgo = new Date().setDate(now.getDate() - 3);
```